cmake/LibraryConfigurations.cmake

```diff
@@ -40,10 +40,18 @@ elseif (${PLATFORM} MATCHES "Android")
     set(PLATFORM_CPP "PLATFORM_ANDROID")
     set(GRAPHICS "GRAPHICS_API_OPENGL_ES2")
     set(CMAKE_POSITION_INDEPENDENT_CODE ON)
-    
+    set(raylib_sources
```

Comment:
It seems `raudio.c` is missing...

Reply:
> It seems `raudio.c` is missing...
yes it's missing but it's missing in the main cmake where I took so It's not from my side

Reply:
so I will push another pull to fix this and the main one

Reply:
I think it should be added and tested.

Reply:
I will add it as option too

Reply:
wait a minute it's there

Reply:
in src/CMakeLists.txt line 45